### PR TITLE
feat: Note well enhancements

### LINKED ISSUE #194: Use https://github.com/ietf/note-well/ for Note Well
IETF Note Well has a new GitHub repository: https://github.com/ietf/note-well/
Wagtail website should get the **Note Well** from this repository.

The new **note well** snippet developed in #30 should make use of https://github.com/ietf/note-well/blob/main/note-well.md to get the content.

### PR DESCRIPTION
**Pull request by AndrewCalderSpringload**
This pull request makes note well text come from the note well page and automatically updates that page with the content of the Note Well repo.

Fixes #194

### CODE REVIEW

File: ietf/snippets/templates/snippets/disclaimer.html [hunk 1]
```diff
@@ -4,29 +4,20 @@
...
+                    {% if note_well %}
+                        <h2>{{ note_well.title }}</h2>
+                        {% if note_well.prepared_key_info %}
+                            {{ note_well.prepared_key_info|safe }}
+                        {% endif %}
+                        {% if note_well.prepared_in_depth %}
+                            {{ note_well.prepared_in_depth|safe }}
+                        {% endif %}
+                    {% else %}
+                        {# Fall back to some default text - copied from production note well at the time of writing #}
```
Comment (kesara) [inline ietf/snippets/templates/snippets/disclaimer.html:17]: Having a fallback is a good thing but I don't think showing a _possibly_ out of date version of note well text is a good idea.
Maybe add a link to the note well repository and contact details to let IETF support knows about the issue?
Something like https://notes.ietf.org/YXWF94BKRH-uqmQfRc-P1Q?view
@ghwood What do you think?

File: ietf/standard/management/commands/update_note_well.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+logger = Logger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Update the note well page from the note well github repo"
+
+    def handle(self, *args, **options):
+        # Get content from github
+        note_well_url = (
+            "https://raw.githubusercontent.com/ietf/note-well/main/note-well.md"
```
Comment (kesara) [inline ietf/standard/management/commands/update_note_well.py:18]: Can you move this out to the SETTINGS?
Also, it's better to use https://github.com/ietf/note-well/raw/main/note-well.md 
That URL redirects to the githubusercontent.com

File: ietf/snippets/templates/snippets/disclaimer.html (hunk 1 above)
Comment (ghwood) [reply ietf/snippets/templates/snippets/disclaimer.html:17]: Yes, I agree a pointer would be better.

File: ietf/standard/management/commands/update_note_well.py (hunk 2 above)
Comment (AndrewCalderSpringload) [reply ietf/standard/management/commands/update_note_well.py]: Done, and done.

**Review by kesara — APPROVED** [review]

**Pull request merged**